pkg/rebuild/rebuild/rebuildremote.go

```diff
@@ -243,6 +310,10 @@ var proxyBuildTpl = template.Must(
 						docker buildx build --builder proxied --build-context certs=/etc/ssl/certs --secret id=PROXYCERT --load --tag=img -
 					{{.Dockerfile}}
 				EOS
+					{{- if .UseSyscallMonitor}}
```

Comment:
let's also move this above the docker build so we capture any activity that happens there